examples/demo_launcher/lib/main.dart

```diff
@@ -181,8 +181,8 @@ class DemoList extends Component {
   }
 }
 
-class SkyHome extends App {
-  Widget build() {
+class SkyHome extends StatelessComponent {
+  Widget build(BuildContext context) {
     return new Theme(
       data: new ThemeData(
         brightness: ThemeBrightness.light,
```

Comment:
take out Theme and Title and make runApp use an App with a '/' route that just builds a SkyHome.
Also rename it FlutterHome.